File: backend/src/HousingApp.Application/Auth/DTOs/RegisterDto.cs
namespace HousingApp.Application.Auth.DTOs
{
    public record RegisterDto(
        string Email,
        string Password,
        string Role,
        string FirstName,
        string LastName,
        string PhoneNumber,
        string Nationality,
        int Age,
        string Gender,
        string ImageUrl,
        string BirthDate
    );
}

File: frontend/src/features/auth/types/registerDto.ts
import { RoleEnum } from "../../../global/enum/role";

export type RegisterDto = {
	email: string;
	password: string;
	role: RoleEnum;
	firstName: string;
	lastName: string;
	phoneNumber: string;
	nationality: string;
	gender: string;
	imageUrl: string;
	birthdate: string;
};
